### PR TITLE
Fix restoration

### PR DESCRIPTION
**Pull request by Josue-T**
# Problem

- The service don't start when the app installed. There are a problem about the permission for the file `/opt/$app/gunicorn_config.py`
- The database is not saved in the backup script and not restored in the restore.
- The dependence in the install script and restore script are different

# Solution

- Remove the helper postgresql. We will use the official. And adapt the code to use to offical.
- Universalize the dependence in all scripts.
- Fix some others small things...

Tested with CI [Build Status/2/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/coin_ynh%20(josue)/2)

### CODE REVIEW

File: scripts/restore [hunk 1]
```diff
@@ -44,6 +44,8 @@ ynh_package_install python3-pip python3-virtualenv
 # Restore sources & data
 sudo cp -a ./sources $final_path
 
+# Create user
+useradd $app -d $final_path || echo "User already exists"
```
Comment (alexAubin) [inline scripts/restore]: Uuuh shouldnt we use the appropriate helper for that ? :P Or did you just copypasta from the install script ?

Comment (Josue-T) [reply scripts/restore]: Just copy past :-)

**Comment by Josue-T** [comment]
Well, I found many other issue in this app.

So I'm improving this PR with some others things

**Comment by Josue-T** [comment]
Now it's ready for a review

**Pull request merged**